templates/lx_userdata.sh

```diff
@@ -258,7 +258,7 @@ install-watchmaker() {
   try_cmd 1 python3 -m pip install --index-url="$PYPI_URL" --upgrade boto3 requests
 
   # Clone watchmaker
-  try_cmd 1 git clone "$GIT_REPO" --recursive
+  try_cmd 3 git clone "$GIT_REPO" --recursive
```

Comment:
This is an improvement, but just nothing that the retries may begin to fail in another way, depending on exactly how far the first attempt got. In particular, if the directory is created by the first attempt, and then the clone fails, the subsequent clones will fail because the directory already exists. Can address that later though. Perhaps by wrapping this in a function that force removes the directory first, and putting the retry on the function.

Reply:
Let me add the directory removal.  Should be straight forward.